connect/middleware.go

```diff
@@ -0,0 +1,28 @@
+package connect
+
+import (
+	"github.com/micromdm/micromdm/device"
+	"golang.org/x/net/context"
+)
+
+type lastCheckinMiddleware struct {
+	devices device.Datastore
+	next    Service
+}
+
+func NewMiddleware(datastore device.Datastore, next Service) (*lastCheckinMiddleware) {
```

Comment:
Your `Acknowledge` function already uses `ackQueryResponses()` which I think is the correct place to persist the data, instead of wrapping it in middleware.  

Is this a holdover from an earlier commit?